### ISSUE TITLE
Couple issues with Photovoltaic Cells

### ISSUE THREAD

**Issue by WindFox** [SleepyTrousers/EnderIO#4970]
#### Issue Description:

Photovoltaic cells report that they cannot see the sun when they can and report that they can when a solid block is placed on top. Also, they don't seem to be actually outputting power.

#### What happens:
When I check the cells with a probe they are listed as generating power but then the probe says that they cannot see the sun. But if I place a block (i.e. cobblestone) in the air right above blocking the light; the probe shows that no energy is being produced but they CAN see the sun (whoops!)

Also, it doesn't seem like they actually have power to send to the conduits below. (do I need a grid of energy conduits or can I just connect to ONE cell in a grid of cells?) I have a small amount of things on my grid currently, the only constant drain is the 5 mI of the power monitor (TIL that has a constant drain), and the probe shows that I am slowly losing energy because of it. I thought that the 280 mI/t at noon is supposed to cover that. And no, I do not have simple machines, and yes, I just take out the monitor the drain will stop.

#### What you expected to happen:
The probe not reversing the sun check on photovoltaic cells, and the cells actually giving power to the grid.

#### Steps to reproduce:

1. Fill energy grid from a generator.
2. Place photovoltaic cells out in the sun connected to the grid
3. Check cells with probe
4. Place block on top of cells and check again.
5. Place power monitor on grid and check grid.

____
#### Affected Versions (Do *not* use "latest"):

- EnderIO: 5.0.38
- EnderCore: 0.5.43
- Minecraft: 1.12.2
- Forge: 14.23.5.2784
- SpongeForge? no
- Optifine? heck no
- Single Player

#### Your most recent log file where the issue was present: 

Can get on request.

As for asking the pack maker.... I AM the pack maker. 😃 
No seriously. It's just a smallish pack with EIO and most mods that a connection to it (TCon, AE2, RS, etc. Just not Forestry). Also a couple of other QoL mods. Can give list on request as well.

**Comment by tyler489** [comment]
Can Confirm with Just Enderio https://gyazo.com/e53f0fa1fa53179d2b479f777cc5794b

**Comment by WindFox** [comment]
Yeah, also just did a test with just EIO and JEI.
Quick question about the other part: is there some sort of priority system with power generators? In my quick test, I noticed that the solar panels would not push power until the sterling generator was completely empty. (Capacitor block was not touched.)

**Comment by MatthiasMann** [comment]
@WindFox use a power monitor to enable/disable the Stirling (or other) generators based on the stored power of your network.
Do not put the capacitor inline - just attach it to the power network like any other machine. That way it will store unused energy and supply extra power when needed without limiting the power flow. As the limits of the energy conduits are based on each individual connection this will give you more available power.

**Comment by HenryLoenwind** [comment]
The order in which energy sources push their energy into the network is random (it's the order Minecraft ticks the blocks). Capacitor banks are an exception here as they regulate (on conduit networks).

**Comment by WindFox** [comment]
@MatthiasMann This is exactly what I did in my pack and my quick test. When I didn't have the monitor in the network and was not using the machines the energy stored in the sterling generator would not move. makes sense since everything was full and nothing was doing anything. However, once I put the monitor on the network, it'll draw 5 mI/t and I noticed that the sterling generator would be the only one to provide that power until it's buffer was empty, only then will the solar kick in to provide power to the grid.

It could be I'm just used to other mods where the passive would be the first place a network would look.
It may also be not as big of a deal than I originally thought since I was trying to use the monitor to save on fuel even before I put solar on my very small network, but once I did put in solar and not initially seeing the draw from it because the generator still had stored power I just freaked out a bit.
